<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dicomifier::gui::PACSConfigurationFrame</class>
 <widget class="QDialog" name="dicomifier::gui::PACSConfigurationFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PACS Configuration</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QPushButton" name="AcceptButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>88</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="CancelButton">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>88</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget_3
{
    border-radius: 8px;
    border: 1px solid #2299FF;
	margin: 5px;
}
QWidget#widget_4
{
    border-radius: 8px;
    border: 1px solid #2299FF;
	margin: 5px;
}
QWidget#widget_5
{
    border-radius: 8px;
    border: 1px solid #2299FF;
	margin: 5px;
}
QWidget#widget_6
{
    border-radius: 8px;
    border: 1px solid #2299FF;
	margin: 5px;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="pacsName">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="0">
          <widget class="QLabel" name="firstTitle">
           <property name="text">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="secondTitle">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="pacsFrist">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Authentication type:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="pacsSecond">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QComboBox" name="pacsIdType">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Address:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="pacsAddress">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="pacsPort">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Called:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="pacsCalled">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Caller:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="pacsCaller">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QPushButton" name="TestButton">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>88</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Test</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>dicomifier::gui::PACSConfigurationFrame</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pacsName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>dicomifier::gui::PACSConfigurationFrame</receiver>
   <slot>onDataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pacsAddress</sender>
   <signal>textChanged(QString)</signal>
   <receiver>dicomifier::gui::PACSConfigurationFrame</receiver>
   <slot>onDataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pacsPort</sender>
   <signal>textChanged(QString)</signal>
   <receiver>dicomifier::gui::PACSConfigurationFrame</receiver>
   <slot>onDataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pacsCalled</sender>
   <signal>textChanged(QString)</signal>
   <receiver>dicomifier::gui::PACSConfigurationFrame</receiver>
   <slot>onDataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pacsCaller</sender>
   <signal>textChanged(QString)</signal>
   <receiver>dicomifier::gui::PACSConfigurationFrame</receiver>
   <slot>onDataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>607</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pacsIdType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>dicomifier::gui::PACSConfigurationFrame</receiver>
   <slot>onDataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>685</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pacsFrist</sender>
   <signal>textChanged(QString)</signal>
   <receiver>dicomifier::gui::PACSConfigurationFrame</receiver>
   <slot>onDataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pacsSecond</sender>
   <signal>textChanged(QString)</signal>
   <receiver>dicomifier::gui::PACSConfigurationFrame</receiver>
   <slot>onDataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>661</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>716</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onDataChanged()</slot>
 </slots>
</ui>
